src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/translation.html.twig

```diff
@@ -57,14 +54,6 @@
 {% endblock %}
 
 {% block panelContent %}
-    {% set filter = request.query.get('state', '-1') %}
```

Comment:
This code is for adding the translation profiler (see # 14993 ), is there a reason to remove it or just by mistake ?